### ISSUE TITLE
I don't know if anyone told you this before, but the 3 dots to minimize, maximize and close the window have very bright colors and it is very distracting. could you tone them down a bit please. It's hard to find a beautiful theme that also helps you be productive.

### ISSUE THREAD

**Issue by AndiHamolli** [EliverLara/Ant-Dracula#34]
I don't know if anyone told you this before, but the 3 dots to minimize, maximize and close the window have very bright colors and it is very distracting when using ant-dracula. could you tone them down a bit please. It's hard to find a beautiful theme that also helps you be productive.

**Comment by joveice** [comment]
I like these colours, also don't think they are that bright, but you could just fork it and change the colour to what you like.

**Comment by vanillajonathan** [comment]
I agree that they are too bright and should be toned down a bit to have less contrast / luminosity.
